### PR TITLE
Redcore 431 Complex type editor fix

### PR DESCRIPTION
**Pull request by Mathewlenning**

### CODE REVIEW

**Comment by Kixo** [comment]
what happened with those minified lib medias?
did you changed anything there?

**Comment by Mathewlenning** [comment]
Wow I'm not sure what happened to the minified media. I did have to run the gulp:scripts once when I was trying to get the changes to redcore.js to take affect. I added the WS record ID to redCORE.ws.addNewRow ajax URL

because I needed to load the xml file, so that the new complex types would be included in the transform options.

File: build/media/libraries/redcore/media/redcore/css/component.bs3.css [hunk 1]
```diff
@@ -9763,7 +9723,7 @@
 .redcore .chart-box {
   padding: 0;
 }
-.redcore .modal-body {
+.redcore .redcore .modal-body {
```
Comment (Kixo) [inline build/media/libraries/redcore/media/redcore/css/component.bs3.css]: this cant be right...

Comment (Kixo) [inline build/media/libraries/redcore/media/redcore/css/component.bs3.css]: can you please fix this one...

**Comment by Kixo** [comment]
@Mathewlenning can you exclude your changes to the minified library files... You have no css changes in this PR, but some library files are being updated, they are minified so it is hard to see what exactly you are changing

**Comment by Mathewlenning** [comment]
I created another PR, so I'm closing this one.

**Pull request closed without merging**